client/src/components/Reviews/components/ReviewList.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, fireEvent, waitFor, screen } from '@testing-library/react';
+import ReviewList from 'Reviews/components/ReviewList.jsx';
+
+describe('test', () => {
+  test('Ratings & Reviews', async () => {
+    const user = userEvent.setup();
+    render(<ReviewList />);
+
+    await user.click(screen.getByText('More reviews'));
+
+    expect(screen.getByText('More reviews')).not.toBeDisabled();
+  });
+  test('Ratings & Reviews helpful', async () => {
+    const user = userEvent.setup();
+    render(<ReviewList />);
+
+    await user.click(screen.getAllByText('| Report')[0]);
+    await user.click(screen.getAllByText('| Report')[1]);
+
+    expect(screen.getAllByText('| Report')[0]).not.toBeDisabled();
```

Comment:
I'm having trouble understanding the functionality this test aims to evaluate. Are you checking that the buttons aren't disabled after clicking on them? Is there a situation where the buttons should be disabled? I think it's useful to lean on the business requirements document as a guide to testing, so that we're limiting our tests to the crucial functionality.

Reply:
Since my buttons don't have the functionality built yet (waiting on others things to be done) I just wanted to try writing some tests just to understand them. ill update the tests once the buttons do more

Reply:
Ok that's what I figured. Looks like your tests are set up well though so you should be good to go.